<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLogisticaAgendarColeta</class>
 <widget class="QWidget" name="WidgetLogisticaAgendarColeta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonSendRight">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonSendLeft">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Agendar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="suffix">
        <string> Kg</string>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="QTreeView" name="tableTransp">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>153</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>153</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>153</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeView::item { border: 0.5px ; border-style: solid ; border-color: lightgray ;}</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QTreeView" name="tableEstoque">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>153</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>153</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>153</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
